React/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+import store from '../../../redux/store';
+
+import LoginForm from './components/LoginForm';
+
+// Separé el container del presentacional. Este es el container se supone
+// El form no me toma los errores. Muestra los mensajes de error pero hasta ahi nomas
+/*
+const Login = ({ history }) => (
+  <LoginForm onSubmit={handleSubmit} />
+);
+*/
+
+class Login extends Component {
+  handleSubmit = () => {
+    store.dispatch({
+      type: 'SET_LOGGED_USER',
+      logged: true
+    });
+    this.props.history.push('/app');
```

Comment:
use route as constant.. and the defaul route should not be /app, it should be `/`